src/pd_density.jl

```diff
@@ -1,9 +1,20 @@
 module pd_density
 
 using Images, Optim
+using FFTW, ZernikePolynomials, DSP
+
+struct InitialParam
+    n::Float64
+    NA::Float64
+    lambda::Float64
+    imsz::Tuple
```

Comment:
```suggestion
    imsz::Tuple{Int,Int,Int}
```
This will always have length 3, right?
```julia
julia> isconcretetype(Tuple)
false
```
and thus not inferrable, but
```julia
julia> isconcretetype(Tuple{Int,Int,Int})
true
```
You don't *always* have to make things inferrable, but when you can it's typically better to do so. (There are specific cases where you deliberately want to make something non-inferrable to prevent compiler specialization, but that's a more advanced design choice.) If this tuple won't always be of length 3, you could make the struct parametric: `struct InitialParam{N}` followed by `imsz::NTuple{N,Int}`.

An alternative, though, is to get this from `img`; see comment below.